components/Venue/venue.js

```diff
@@ -1,30 +1,34 @@
 /* eslint-disable react/no-unescaped-entities */
 import React from 'react';
+import Link from 'next/link';
 
 function Venue({ className, city }) {
 	return (
-		<div>
-			<div style={{'--image-url': `url(${city.img})`}} 
-			className='relative w-[300px] h-[400px] sm:w-[250px] sm:h-[350px] card-bg bg-[image:var(--image-url)] flex items-center justify-center p-4 cursor-pointer'>
+		<Link href={`/venue/${city.name}`}>
+			<div style={{'--image-url': `url(${city.img})`}}
+				className='relative w-[300px] h-[400px] sm:w-[250px] sm:h-[350px] card-bg bg-[image:var(--image-url)] flex items-center justify-center p-4 cursor-pointer'>
 				<div className='flex justify-between flex-col w-full h-full'>
 					<div className='flex justify-between items-center'>
 						<div><div className={`border text-white text-md rounded-lg p-1 text-center mt-2 ${city.cfp ? 'block': 'hidden' }`}>cfp is open</div></div>
-						<a href={city.map} target='_blank' className='w-8 h-8 bg-white rounded-xl flex items-center justify-center' rel="noreferrer">
-						<img src='/img/mapIcon.svg' className='w-6' />
-						</a>
+						<button onClick={(e) => {
+							e.preventDefault()
+							window.open(city.map, '_blank')
```

Comment:
Can you make the window.open function look like this,` (city.map,'_blank','noopener')`? cuz of vulnerability issue